extensions/korean-date-converter/README.md

```diff
@@ -0,0 +1,3 @@
+# Korean Date Converter
+
+한국어로 된 상대적인 날짜와 시간 표현을 날짜로 변경해줍니다. 이 라이브러리를 사용하면 "다음주 화요일", "3일 후",  등의 다양한 한국어 표현을 날짜로 쉽게 변환할 수 있습니다.
```

Comment:
Could you add this in English instead? You can create a README-ko.md if needed with this.

Reply:
Thank you. I worked on it. It does not need README-ko.md.